mlrun/runtimes/nuclio/application/application.py

```diff
@@ -361,13 +365,14 @@ def create_api_gateway(
         ports: list[int] = None,
         authentication_mode: schemas.APIGatewayAuthenticationMode = None,
         authentication_creds: tuple[str] = None,
+        ssl_redirect: bool = False,
     ):
         api_gateway = APIGateway(
             APIGatewayMetadata(
                 name=name,
                 namespace=self.metadata.namespace,
                 labels=self.metadata.labels,
-                annotations=self.metadata.annotations,
+                annotations=self.metadata.annotations or {},
```

Comment:
We already set it to `{}` [here](https://github.com/TomerShor/mlrun/blob/15a82a586d8e680983d73122fcdbeb0c4c7de350/mlrun/runtimes/nuclio/api_gateway.py#L135)

Reply:
Missed that, fixed.